CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### 🐞 Bug fixes
 
+* Add source field requirement to terrain exaggeration ([#11664](https://github.com/mapbox/mapbox-gl-js/pull/11664))
```

Comment:
@tristen Could you remove that from the actual changelog file and add it to the PR description? 

Our process for changelog is to compile it before the release. And we have a CI check that looks on the description and fails if it's not marked with the label 'skip changelog'

<img width="866" alt="Screen Shot 2022-04-04 at 1 47 41 PM" src="https://user-images.githubusercontent.com/7061573/161629431-10bf34cb-35cf-460e-ba09-e1c26b48327c.png">

PR looks good otherwise!